app/com/arpnetworking/metrics/portal/alerts/impl/DailyPartitionCreator.java

```diff
@@ -303,7 +303,7 @@ protected void execute(
         // it still returns a "non-empty" void result and so we can't use the
         // ordinarily more appropriate SqlUpdate type.
         final SqlQuery sql = _ebeanServer.createSqlQuery(
-                "select * from create_daily_partition(?::text, ?::text, ?::date, ?::date)")
+                "select * from portal.create_daily_partition(?::text, ?::text, ?::date, ?::date)")
```

Comment:
I'm not sure why this passes integration tests, still trying to figure that out. It's likely because the actor does not report any problems in a way the test could catch.

I manually created the partitions for the next 2 weeks to mitigate this issue until this fix is out.